src/Modal/Title.js

```diff
@@ -0,0 +1,24 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import css from 'styled-jsx/css'
+import cx from 'classnames'
+
+export const Title = ({ children }) => (
+    <h1 className={cx('title')}>
```

Comment:
Actually I wanted to make the `Modal` wrapper an `aside`.. Forgot about that one.
In terms of semantic, I disagree with our competitors (when using `article`) unless we'd be supporting HTML4 browsers with SEO.

* The Modal is not part of the document's main flow (it's already at the end of the `document`) and therefore has no relation to other headlines.
* Given the above is true, h5/h6 indicate that there are higher level headlines (which there are not)
* Using h5/h6 doesn't really make sense if there's no h4/h5
* If we wanted the headline to align with the content that triggered the `Modal` to open, the headline would need to be dynamic as we don't know which headline that content is using (which I'd only agree to if we were doing SEO, because this is quite a hassle and overcomplicating things)

I'd prefer to change the `Modal` to an `aside` element, which is a sectioning element that allows `h1` as well, [see the first w3c example, which includes an h1 as well](https://www.w3.org/TR/2014/REC-html5-20141028/sections.html#the-aside-element).

Reply:
Thanks for explaining your motivations for making this a h1. I'm convinced.